src/components/AppBar/AppBar.js

```diff
@@ -85,19 +79,18 @@ const NavBar = () => {
               <>
                 <Drawer />
                 <IconButton onClick={() => dispatch(openDrawer())}>
-                  <span style={{ position: 'absolute', top: 0, right: '8px', color: 'red', fontSize: '16px' }}>
-                    {cartItemCount === 0 ? null : cartItemCount}
-                  </span>
+                  {cartItemCount > 0 && (
+                    <CartItemQuantity>{cartItemCount === 0 ? null : cartItemCount}</CartItemQuantity>
+                  )}
                   <ShoppingCartIcon />
                 </IconButton>
               </>
             )}
-
             {isLoading ? (
               <CircularProgress />
             ) : (
               <>
-                <UserName>{name}</UserName>
+                <UserName>{name.indexOf(' ') >= 0 ? name.substr(0, name.indexOf(' ')) : name}</UserName>
```

Comment:
Can't we use something like split and take the first array element ..
`const firstname = username.split(' ')[0]`;